targets/wasm_exec.js

```diff
@@ -407,9 +407,9 @@
 					},
 
 					// func valueInstanceOf(v ref, t ref) bool
-					//"syscall/js.valueInstanceOf": (sp) => {
-					//	mem().setUint8(sp + 24, loadValue(sp + 8) instanceof loadValue(sp + 16));
-					//},
+					"syscall/js.valueInstanceOf": (sp) => {
+ 						return loadValue(sp) instanceof loadValue(sp + 8);
+					},
```

Comment:
I do not think this is correct. Have you tested the code?

The calling convention is different in TinyGo, so it should probably be something like this:

```suggestion
					"syscall/js.valueInstanceOf": (vPtr, tPtr) => {
 						return loadValue(vPtr) instanceof loadValue(tPtr);
					},
```

Although maybe this was already the layout in memory by chance.

Reply:
Yes, I have tested the code. 
But the second argument may have value and I didn't check for it. The offset in TinyGo is `8` unit, but we should use the second argument instead of offsetting it by manual.

Reply:
I will update it later.